libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
```

Comment:
I think you mean the opposite. Leaving `no-dependencies` and removing `--enable-dependency-tracking` from the Prepare script.

You see, dependency-tracking never worked. A no-dependencies option in `Makefile.am.common` preventing it. Now that you migrated `no-dependencies` over to configure.ac dependency-tracking still doesn't work, but at least it tells you right from the beginning.

If you remove `no-dependencies` then

1. the build will be slower
2. everything will be rebuild from scratch every time you run `make`
3. there will be a lot of additional files laying around in your working directory

Dependency-tracking may be useful in some scenarios, but no one in the history of libnet has ever needed it nor has ever complained that it isn't working. We should just drop it from the `Prepare `script to make it official.